src/components/videoCard/buttons/CopyVideoLinkButton.test.tsx

```diff
@@ -64,5 +67,8 @@ describe('CopyLinkButton', () => {
     expect(navigator.clipboard.writeText).toHaveBeenCalledWith(
       buildVideoDetailsLink(video, user),
     );
+    const notification = wrapper.getByTestId('video-link-copied-notification');
+    expect(notification).toBeVisible();
+    expect(notification.children.item(0).textContent).toEqual('Link copied!');
```

Comment:
this one is very specific. We could get it simply by `screen.getByText('Link copied!')` and than assert that it's visible/in the document

Reply:
Nice catch! Also if your reason for using children here was to check the contents of the notification you can use [within](https://testing-library.com/docs/dom-testing-library/api-within/) - `within(notification).getByText('Link copied!')`